package/MDAnalysis/topology/tables.py

```diff
@@ -89,75 +89,110 @@ def kv2dict(s, convertor=str):
 # atomname   element
 # --------- ------------------
 
+# Aluminium
```

Comment:
I'm not sure here, there have been attempts at updating this table in the past and there have been plenty of arguments for/against. Possibly @orbeckst is the best person to say if this is a good idea / won't break anything else.

One possible alternative here is to add an additional dictionary entry to MDAnalysis.topology._elements.py which just reverses `NUMBER_TO_ELEMENTS`. This would then provide a complete match to the entire periodic table rather than only accepting a subset of it (if anything Cobalt is a frequent ion in the PDB and as far as I can see it isn't listed here).

Reply:
Sure, that sounds like a better alternative. I was a bit concerned about making changes in this table.Should I go ahead and 1. Change this dictionary back to original, and create an additional dictionary entry in MDAnalysis.topology._elements.py to reverse it like you said?

Reply:
Might be best to see what any of the others have to say about this before you make any further changes to this table. There's a good chance there's a very simple solution here that I'm just not seeing.

Reply:
Oh okay. Definitely. I would just update the remaining changes in the attributes for now then...

Reply:
Just popping in to point out the `SYMB2Z` and `Z2SYMB` dictionaries in `tables.py`, which feature more modern element names.

Also, yeah, this is a really good example of how hard it is to guess anything with names. `elements` is so short because of MD systems where an atom called '1he2' is more probably hydrogen than helium, and so on. This is probably better discussed in #2630.

Anyway you're just validating element symbols so `SYMB2Z` should do.

Reply:
I knew I was probably missing something obvious 😅 Thanks @lilyminium, although, if I'm getting this right, we have a whole extra file that just duplicates `Z2SYMB` (except that `NUMBER_TO_ELEMENT` contains pre-2016 IUPAC names)?

Reply:
I suppose so! Although more like `Z2SYMB` duplicates `NUMBER_TO_ELEMENT` due to me not realising the latter existed until you mentioned it just now. `grep` says it's only used for `TOPParser`.

Reply:
Just to ensure that I am on the same page before I add it to the guess_atom_element option, I think the discussion above primarily implies to do something as follows:

```
if name in tables.atomelements:
            return Z2SYMB[SYMB2Z[tables.atomelements[name].lower().capitalize()]]
elif name in tables.elements:
                return Z2SYMB[SYMB2Z[name.lower().capitalize()]]

```